app/Http/Controllers/HomeController.php

```diff
@@ -26,7 +26,7 @@ public function index(Son $son)
     {
         $sons = [];
         if (auth()->user()->hasRole('admin|pmo')) {
-            $sons = $son->sortable()->paginate(50);
+            $sons = $son->sortable(['title' => 'asc', 'updated_at' => 'desc'])->paginate(50);
```

Comment:
I think we want the most recent at the top?
```suggestion
            $sons = $son->sortable(['updated_at' => 'desc', 'title' => 'asc'])->paginate(50);
```

Reply:
yeah this is really up to how shannon wants it.